cmd/mount.go

```diff
@@ -509,7 +534,7 @@ func startMonitor(pid int) {
 	}
 }
 
-func sigusrHandler(pipeline *internal.Pipeline, ctx context.Context) daemon.SignalHandlerFunc {
+func sigusrHandler() daemon.SignalHandlerFunc {
```

Comment:
pipeline and context are needed here. Context is taken for future use where we can terminate the ongoing processing.

Reply:
Seems like they are not used in this function. That's why I deleted them.

Reply:
kept for future use so lets keep them as is.

Reply:
ok, will make no difference